tools/src/main/java/org/apache/rocketmq/tools/command/MQAdminStartup.java

```diff
@@ -194,7 +197,7 @@ public static void initCommand() {
     private static void initLogback() throws JoranException {
         String rocketmqHome = System.getProperty(MixAll.ROCKETMQ_HOME_PROPERTY, System.getenv(MixAll.ROCKETMQ_HOME_ENV));
 
-        LoggerContext lc = (LoggerContext) LoggerFactory.getILoggerFactory();
+        LoggerContext lc = (LoggerContext)LoggerFactory.getILoggerFactory();
```

Comment:
Also please follow the RocketMQ [code-style](http://rocketmq.incubator.apache.org/docs/code-guidelines/)